src/pages/Login/LoginInput.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styled from "styled-components";
+
+const LoginInput = (props) => {
+  const { text } = props;
+  return <InputBox placeholder={text} />;
+};
```

Comment:
```suggestion
const LoginInput = ({text}) => {
  return <InputBox placeholder={text} />;
};
```
구조분해할당은 인자에서 이렇게 바로 해줄 수 있습니다!

Reply:
네 반영했습니다!